invenio_search_ui/assets/semantic-ui/js/invenio_search_ui/components/MultipleOptionsSearchBar.js

```diff
@@ -170,6 +175,7 @@ MultipleOptionsSearchBarCmp.propTypes = {
 
 MultipleOptionsSearchBarCmp.defaultProps = {
   placeholder: i18next.t("Search records..."),
+  ariaLabel: i18next.t("Search"),
```

Comment:
I would say it could be hardcoded... unless there is a reason for it to be a prop @lhenze

Reply:
That is how we've done it everywhere else, to my knowledge, so, consistency would make sense. That said, I don't feel strongly about it; it's your call.

Reply:
I would prefer not to add more complexity to the components' interfaces, if we can hardcode the value, we should

Reply:
If you have 2 `MultipleSearchBar` components in the same page, I guess you will need to define different area label, no? @lhenze @jennur ...If yes, then we can keep it like this otherwise we can start the habit of not exposing the `areaLabel` to the component's interface and hardcode it so we have full control of it!

Reply:
The aria-label should just describe what the button does, so it doesn't have to be unique. For the search bar, the button should always trigger a search, so I don't think it will ever be needed to customize the label in this case.

Reply:
So, to clarify you are also leaning towards hardcoding the `areaLabel` value right @jennur ?

Reply:
Yes, I don't see the need to have it be customizable...